### PR TITLE
Replace any occurrence of yum-centos/yum-epel/yum-elrepo with osl-repos equivalents

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -17,8 +17,8 @@
 # limitations under the License.
 node.default['yum']['powertools']['enabled'] = true
 node.default['yum']['powertools']['managed'] = true
```
Comment (ramereth) [inline recipes/default.rb:19]: This should be replaced with using the `edit_resource` method now.

Comment (TheREK3R) [reply recipes/default.rb:19]: I actually just forgot to remove this, powertools is installed by default in osl-repos bc it was in the base recipe

**Comment by ramereth** [comment]
LGTM

**Review by ramereth — APPROVED** [review]
Please do a minor bump on `~`

**Comment by TheREK3R** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v2.10.0.  Have a nice day!

**Pull request merged**